contribs/gmf/examples/style/mobileapp-map.less

```diff
@@ -7,3 +42,144 @@ gmf-map {
     left: 0;
   }
 }
+
+.ol-zoom {
+  background-color: transparent;
+  right: 0;
+  left: auto;
+  top: 52px;
+  margin: 10px;
+  button {
+    margin-bottom: 3px;
+    height: @map-tools-size;
+    width: @map-tools-size;
+    background-color: @map-tools-bg-color;
+    border: solid 1px @map-tools-color;
+    color: @map-tools-color;
+    &:hover {
+      background-color: @onhover-color;
+    }
+  }
+  &:hover {
+    background-color: transparent;
+  }
+}
+
+.ol-attribution {
+  display:none;
+}
+
+.ol-scale-line, .ol-scale-line-inner {
+  background-color: @map-tools-bg-color;
+  border-color: @map-tools-color;
+  border-width: 2px;
+  color: @map-tools-color;
+}
+
+.geolocation {
+  right: 0;
+  left: auto;
+  top: 137px;
+  margin-right: @app-margin;
+  z-index: @above-content-index;
+  &:after {
+    content: 'b';
+  }
+}
+
+
+.search {
+  top: @app-margin;
+  left: 0;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 6px 10px;
+  background-color: @map-tools-bg-color;
+  width: 162px + @map-tools-size + @map-tools-size;
+  border-radius: 25px;
+  border: solid 1px @map-tools-color;
+  z-index: @content-index;
+  input {
+    width: 150px;
+    margin: 0 10px;
+    padding: 0 5px;
+    border-width: 0 1px;
+    border-radius: 0;
+  }
+}
+
+.nav-trigger {
+  top: @app-margin;
+  left: 50vw;
+  margin-top: 2px;
+  background-color: transparent;
+  border: none;
+  z-index: @above-content-index;
+}
+
+.nav-left-trigger {
+  margin-left: -102px;
+  &:after {
+    content: 'a';
+  }
+}
+
+.nav-right-trigger {
+  margin-left: 77px;
+  &:after {
+    content: 'd';
+  }
+}
+
+//For tablet only
+@media (min-width: @screen-sm-min) {
+  .ol-zoom {
+    top: auto;
+    bottom: 36px;
+  }
+
+  .geolocation {
+    top: auto;
+    bottom: 0px;
+    margin-bottom: 10px;
```

Comment:
This is because you already set "top: 137px" for all medias.
I think it would be better to add a new media query for media with `max-width: @screen-sm-max`.